compiler/test/suites/stdlib.re

```diff
@@ -14,45 +14,75 @@ describe("stdlib", ({test, testSkip}) => {
   assertSnapshot("stdlib_cons", "[1, 2, 3]");
   assertSnapshot(
     "stdlib_equal_1",
-    "import * from \"list\"; (1, 2) is (1, 2)",
+    "include \"list\" as List; from List use *; (1, 2) is (1, 2)",
   );
   assertSnapshot(
     "stdlib_equal_2",
-    "import * from \"pervasives\"; (1, 2) == (1, 2)",
+    "include \"pervasives\" as Pervasives; from Pervasives use *; (1, 2) == (1, 2)",
   );
   assertSnapshot(
     "stdlib_equal_3",
-    "import * from \"list\"; [1, 2, 3] == [1, 2, 3]",
+    "include \"list\" as List; from List use *; [1, 2, 3] == [1, 2, 3]",
+  );
+  assertSnapshot(
+    "stdlib_equal_4",
+    "include \"list\" as List; from List use *; 1 == 1",
+  );
+  assertSnapshot(
+    "stdlib_equal_5",
+    "include \"list\" as List; from List use *; 1 == 2",
+  );
+  assertSnapshot(
+    "stdlib_equal_6",
+    "include \"list\" as List; from List use *; true == true",
+  );
+  assertSnapshot(
+    "stdlib_equal_7",
+    "include \"list\" as List; from List use *; true == false",
+  );
+  assertSnapshot(
+    "stdlib_equal_8",
+    "include \"list\" as List; from List use *; [>] == [>]",
+  );
+  assertSnapshot(
+    "stdlib_equal_9",
+    "include \"list\" as List; from List use *; [>] == [> 1]",
+  );
```

Comment:
Why do all these include list?

Reply:
I think it was to test that importing a library works? We can almostly certainly refactor these (not that they test anything anyway, since the entire program should be optimized away). But different PR 😛

Reply:
Please open an issue 🙏

Reply:
https://github.com/grain-lang/grain/issues/1631